ota-plus-web/app/reactapp/src/partials/OperationCompletedInfo.jsx

```diff
@@ -33,17 +34,19 @@ class OperationCompletedInfo extends Component {
   }
 
   render() {
-    const { info } = this.props;
+    const { info, notDisappear } = this.props;
```

Comment:
Could you rename `notDisappear` to `preserve` maybe?

Reply:
Done